gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/LinkerConstants.java

```diff
@@ -0,0 +1,4 @@
+package com.ca.apim.gateway.cagatewayexport.tasks.explode.linker;
+
+public class LinkerConstants {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this empty class, write its code or make it an "interface". ([squid:S2094](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2094))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWgqdHeOSi5GWQS9qtJF&open=AWgqdHeOSi5GWQS9qtJF)

[ISSUE_KEY]: AWgqdHeOSi5GWQS9qtJF